smithy-model/src/main/java/software/amazon/smithy/model/validation/ValidationUtils.java

```diff
@@ -80,21 +86,10 @@ public static String tickedList(Collection<?> values) {
     }
 
     public static String tickedList(Stream<?> values) {
-        return tickedList(values.map(Object::toString).sorted().collect(Collectors.toList()));
+        return values.map(Object::toString).sorted().collect(Collectors.joining("`, `", "`", "`"));
     }
 
-    /**
-     * Find shape IDs in a collection that do not have case-insensitively
-     * unique member names.
-     *
-     * <p>The returned map is sorted by the conflicting name, and the list of
-     * conflicting shape values is also sorted to ensure that validation
-     * events can just print the map directly with consistent error messages.
-     *
-     * @param shapes Shapes collection to check.
-     * @param <T> The type of shape being checked.
-     * @return Returns a map that only contains the conflicting shapes.
-     */
+    @Deprecated
```

Comment:
Has this been unreferenced the whole time? It doesn't look like this commit removes usage of it, but it's being deprecated.